app/controllers/orders_controller.rb

```diff
@@ -34,6 +34,16 @@ def create
     redirect_to root_path
   end
 
+  def show
+    @order = Order.find_by id: params[:id]
+    @order_details = @order.all_details_order
```

Comment:
```suggestion
    @order_details = @order.details_orders
```
chỗ này sẽ bị `n+1` query vì detail sẽ cần gọi tên sản phẩm

tìm hiểu `N+1 query ` và `includes` đi